Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
But now I was thinking. We are leaving all the transitions to the TransitionDelegate, even from `LogIn` to `SignUp`, so maybe we could leave the transition from the LogIn to the main screen of the app (for cases were maybe the user of the framework wants to send the user when logged in or signup to a welcome page and then to the main screen, or I don't know). And instead of making the bootstrapper hear the `SessionService`'s events to act, make it implement a method `userDidLogInSuccessfully` from the TransitionDelegate protocol.